py/test/selenium/webdriver/common/print_pdf_tests.py

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import pytest
+
+from selenium.webdriver import Chrome
+from selenium.webdriver import Firefox
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.options import PrintOptions
+
+start_index = 0
+end_index = 5
+pdf_magic_number = 'JVBER'
+options = Options()
+options.add_argument("--headless") # chrome needs to be headless, print works only in headless
+
+chrome_driver = Chrome(options=options)
+firefox_driver = Firefox()
+
+def test_pdf_with_2_pages(pages):
```

Comment:
Tests should be succinct, these tests appear to have had the kitchen sink thrown in.  We don't test 2 browsers at the same time in any tests so that needs to be rewritten

Reply:
All the tests in this file need to be redone as mentioned above.